### ISSUE TITLE
https://iativalidator.iatistandard.org/ no longer accessible

### ISSUE THREAD

**Issue by siemvaessen** [IATI/IATI-Validator-Actual#37]
**Describe the bug**
Unable to go to https://iativalidator.iatistandard.org/, a password is required?

**To Reproduce**
1. Go to 'https://iativalidator.iatistandard.org/'
2. A modal tells me to enter a passphrase.

**Expected behavior**
https://iativalidator.iatistandard.org/ being accessible to anyone on the web

**Screenshots**
![Screenshot 2021-09-21 at 13 42 17](https://user-images.githubusercontent.com/961036/134164507-6aea044f-cb35-4306-bab5-e72d929469f8.png)

**Comment by alexlydiate** [comment]
Yes, apologies - this was a brief interlude during an update. Now resolved.
